<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QWidget" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLabel" name="labelView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>431</width>
        <height>280</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="sliderScale">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>800</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonOriginalSize">
       <property name="toolTip">
        <string>Reset the scaling to match the original size of the image.</string>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons/view</normaloff>:/icons/view</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonFitToWindow">
       <property name="toolTip">
        <string>Set the scaling to match the size of the viewport.</string>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons/monitor_link</normaloff>:/icons/monitor_link</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionPrintImage">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/print</normaloff>:/icons/print</iconset>
   </property>
   <property name="text">
    <string>Print...</string>
   </property>
   <property name="toolTip">
    <string>Open preview dialog for printing displayed image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionPrintImage</sender>
   <signal>triggered()</signal>
   <receiver>ImageViewer</receiver>
   <slot>openPrintImageDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderScale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImageViewer</receiver>
   <slot>scaleImage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonFitToWindow</sender>
   <signal>toggled(bool)</signal>
   <receiver>ImageViewer</receiver>
   <slot>scaleToFitWindow(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scaleImage(int)</slot>
  <slot>scaleToFitWindow(bool)</slot>
 </slots>
</ui>
